sample-code/spring-app/src/main/resources/static/index.html

```diff
@@ -450,6 +450,15 @@ <h5 class="mb-1">Orchestration Integration</h5>
                                 </div>
                             </div>
                         </li>
+                        <li class="list-group-item">
+                            <div class="info-tooltip">
+                                <a class="link-offset-2-hover link-underline link-underline-opacity-0 link-underline-opacity-75-hover endpoint"
+                                   href="/spring-ai-orchestration/streamChatCompletion"><code>/spring-ai-orchestration/streamChatCompletion</code></a>
+                                <div class="tooltip-content">
+                                    Chat request to OpenAI.
+                                </div>
+                            </div>
+                        </li>
```

Comment:
The website doesn't work, but `localhost:8080/spring-ai-orchestration/streamChatCompletion` does.
We have to remove the javascript from this html page